src/views/tutorial-view/components/image-config/README.md

```diff
@@ -0,0 +1,26 @@
+# ImageConfig
```

Comment:
I'd love to get this into Swingset, but it seems like we're out of luck unless we take all components out of the `views` folder and have them flat in the `components` folder. This felt like too big of a compromise for the moment.